ios/RNInstabug/InstabugReactBridge.h

```diff
@@ -49,7 +49,7 @@
 
 - (void)setString:(NSString *)value toKey:(NSString *)key;
 
-- (void)identifyUser:(NSString *)email name:(NSString *)name;
+- (void)identifyUser:(NSString *)email name:(NSString *)name userId:(nullable NSString *)id;
```

Comment:
Let's change `id` here to `userId` as well.

```suggestion
- (void)identifyUser:(NSString *)email name:(NSString *)name userId:(nullable NSString *)userId;
```

Reply:
Thanks for catching this 🙏